NEWS.md

```diff
@@ -0,0 +1,300 @@
+# 1.7
+
+<!--
+TODO:
+- reword things
+- thank more ppl?
+- sub/3 fixes?
+- validate module metadata is object fix?
+- fix number lexer to avoid conflict with object indexing (#2254)
+- mention some build system changes?
+-->
+
+Lots of JSON has been queried and transformed since the stable release 5 years ago, so now we're happy to finally do a new stable release!
```

Comment:
Was thinking maybe nice to start with some kind summary/introduction like this? what do ppl think? more/less formal?

Reply:
Maybe a bit less formal, yeah.  "After a five year hiatus we're back with a GitHub organization, with new admins and new maintainers who have brought a great deal of energy to make a long-awaited and long-needed new release."?

Reply:
Yeap better